### ISSUE TITLE
Encoding parameter only for reading, not for writing

### ISSUE THREAD

**Issue by alper** [reubano/csv2ofx#27]
I think the encoding parameter should only be used to read in files. Anything produced should be in utf-8 regardless.

I have to set the parameter because my bank returns files in `windows-1252` but I have zero interest in producing a file in that same format. I think that may be the case for more users.

Alternatively, there could be a new parameter to override the output encoding.

**Comment by reubano** [comment]
Good point! Not sure when I'll have the time to address this. PRs welcome of course :).
